modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -3,87 +3,91 @@
 +
 [source,ini]
 ----
 exclude-nics=
 ----
 +
-This entry ensures the following:
+This entry ensures the cloned vNIC generated on the Windows VM by the hybrid-overlay is not ignored.
+
+.. The Windows VM has a valid IP address in vCenter:
++
+[source,posh]
+----
+C:\> ipconfig
+----
+
+.. The VMTools Windows service is running:
++
+[source,posh]
+----
+C:\> Get-Service -Name VMTools | Select Status, StartType
+----
+
+. Install and configure the OpenSSH Server on the Windows VM. See Microsoft's documentation on link:https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse[installing OpenSSH] for more details.
+
+. Set up SSH access for an administrative user. See Microsoft's documentation on the link:https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_keymanagement#administrative-user[Administrative user] to do this. The public key used in the instructions must correspond to the private key you created in the WMCO namespace that holds your secret.
+
+. Install the `docker` container runtime on your Windows VM following the link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server[Microsoft documentation].
+
+. You must create a new firewall rule in the Windows VM that allows incoming connections for container logs. Run the following PowerShell command to create the firewall rule on TCP port 10250:
 +
-* The cloned vNIC generated on the Windows VM by the hybrid-overlay is not ignored.
-* The VM has an IP address in vCenter.
+[source,posh]
+----
+C:\> New-NetFirewallRule -DisplayName "ContainerLogsPort" -LocalPort 10250 -Enabled True -Direction Inbound -Protocol TCP -Action Allow -EdgeTraversalPolicy Allow
+----
 
-.. The VMTools Windows service is running.
+. Create the Windows VM as a reusable image.
 
-. Pull all of the required Windows container base images needed for your applications. The images you pull
-are dependent on the Windows kernel you are using. See Microsoft's documentation on link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/container-base-images[pulling Windows container base images] for more information.
+.. Clone the Windows VM. Follow VMware's documentation on how to link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-1E185A80-0B97-4B46-A32B-3EF8F309BEED.html[clone an existing virtual machine] for more details.
 
-. Run the link:https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation[Windows Sysprep tool] on the Windows VM:
+.. In the cloned Windows VM, run the link:https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation[Windows Sysprep tool] on the Windows VM:
 +
-[source,terminal]
+[source,posh]
 ----
 C:\> sysprep.exe /generalize /oobe /shutdown /unattend:<path_to_unattend.xml>
```

Comment:
Looks like `sysprep.exe` is not recognized,  it works with full path `C:\Windows\System32\Sysprep\sysprep.exe`
```
PS C:\> sysprep.exe /generalize /oobe /shutdown /unattend:unattend.xml
sysprep.exe : The term 'sysprep.exe' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is     
correct and try again.
At line:1 char:1
+ sysprep.exe /generalize /oobe /shutdown /unattend:unattend.xml
+ ~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (sysprep.exe:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```
```
PS C:\> C:\Windows\System32\Sysprep\sysprep.exe /generalize /oobe /shutdown /unattend:unattend.xml
PS C:\> 
```